src/client/components/features/FormationInitiale/Rechercher/RechercherFormationInitiale.test.tsx

```diff
@@ -73,15 +73,53 @@ describe('RechercherFormationInitiale', () => {
 		});
 		it('lorsqu‘il y a un résultat je vois le nombre de résultats affiché au singulier', async () => {
 			const aFormationService = aFormationInitialeService();
-			const resultRechercheFormation = createSuccess([aResultatListFormationInitiale({ libelle: 'boulanger' })]);
+			const resultRechercheFormation = createSuccess([aFormationInitiale({ libelle: 'boulanger' })]);
 			jest.spyOn(aFormationService, 'rechercherFormationInitiale').mockResolvedValue(resultRechercheFormation);
 
 			render(<DependenciesProvider formationInitialeService={aFormationService}>
 				<RechercherFormationInitiale/>
 			</DependenciesProvider>,
 			);
-
 			expect(await screen.findByText(/1 formation/)).toBeVisible();
 		});
+		describe('Lorsque je fais une recherche de formation initale par mot-clé',()=>{
+			it('lorsqu‘il y a des résultats doit affiché le bon nombre de cards',async ()=>{
+				const aFormationService = aFormationInitialeService();
+				const resultRechercheFormation = createSuccess([aFormationInitiale({ libelle: 'boulanger' })]);
+				jest.spyOn(aFormationService,'rechercherFormationInitiale').mockResolvedValueOnce(resultRechercheFormation);
+				render(<DependenciesProvider formationInitialeService={aFormationService}>
+					<RechercherFormationInitiale/>
+				</DependenciesProvider>,
+				);
+				const listeCards =await screen.findByRole('list',{ name: 'Formations Initiales' });
+				const cardTitles=within(listeCards).queryAllByRole('heading',{ level:3 });
+				expect(cardTitles).toHaveLength(1);
+				cardTitles.forEach((item)=>{
```

Comment:
Je te propose cette implémentation, ce test veut checker le nombre de carte visible, c'est un peu dommage de prendre le cas particulier de 1 carte et le fait de faire un 'forEach' sur une liste de 1 élément c'est pas l'idéal 
Dis moi ce que tu en penses :) ?

`			it('lorsqu‘il y a des résultats doit affiché le bon nombre de cards', async () => {
				const aFormationService = aFormationInitialeService();
				const resultRechercheFormation = createSuccess([aFormationInitiale({ libelle: 'boulanger' }), aFormationInitiale({ libelle: 'patissier' })]);
				jest.spyOn(aFormationService, 'rechercherFormationInitiale').mockResolvedValueOnce(resultRechercheFormation);
				render(<DependenciesProvider formationInitialeService={aFormationService}>
					<RechercherFormationInitiale/>
				</DependenciesProvider>,
				);
				const listeCards = await screen.findByRole('list', { name: 'Formations Initiales' });
				const cardTitles = within(listeCards).getAllByRole('heading', { level: 3 });
				expect(cardTitles).toHaveLength(2);
				expect(cardTitles[0]).toBeVisible();
				expect(cardTitles[0]).toHaveTextContent('boulanger');
				expect(cardTitles[1]).toBeVisible();
				expect(cardTitles[1]).toHaveTextContent('patissier');
			});`